src/GitVersionTask/NugetAssets/build/Infrastructure.props

```diff
@@ -9,22 +9,21 @@
     <GitVersionTaskBuildTools_FunctionalityObjFolder>$(GitVersionTaskBuildTools_FunctionalityDir)obj/</GitVersionTaskBuildTools_FunctionalityObjFolder>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(UtilPackNuGetMSBuildAssemblyPath)' == ''">
-    <UtilPackVersion>$(PackageVersion_UtilPackNuGetMSBuild)</UtilPackVersion>
-    <UtilPackNuGetMSBuildPropsPath Condition=" '$(UtilPackNuGetMSBuildPropsPath)' == ''">$(MSBuildThisFileDirectory)../../../utilpack.nuget.msbuild/$(UtilPackVersion)/build/UtilPack.NuGet.MSBuild.props</UtilPackNuGetMSBuildPropsPath>
+  <PropertyGroup Condition=" '$(NuGetUtilsMSBuildExecAssemblyPath)' == ''">
+    <UtilPackVersion>$(PackageVersion_NuGetUtilsMSBuildExec)</UtilPackVersion>
```

Comment:
`UtilPackVersion` should be renamed to `NuGetUtilsVersion` to be consistent with other renames

Reply:
Replaced all remaining UtilPack references with NuGetUtils references now!